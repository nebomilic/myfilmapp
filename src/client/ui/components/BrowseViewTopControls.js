import React from 'react';
const BrowseViewTopControls = () => { 
    return (
        <div className='top-controls'>
            <select name='category'>
                <option value='all'>All</option>
                <option value='comedy'>Comedy</option>
                <option value='action'>Action</option>
                <option value='horror'>Horror</option>
                <option value='scifi'>Sci-Fi</option>
                <option value='drama'>Drama</option>
            </select>
        </div>  

    ); 
};   
export default BrowseViewTopControls;