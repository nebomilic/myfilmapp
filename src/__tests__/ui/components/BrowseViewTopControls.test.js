import React from 'react';
import BrowseViewTopControls from '../../../client/ui/components/BrowseViewTopControls';
import renderer from 'react-test-renderer';

it ('renders corectly', () => {
    const tree = renderer
        .create(<BrowseViewTopControls/>)
        .toJSON();

    expect(tree).toMatchSnapshot();
})