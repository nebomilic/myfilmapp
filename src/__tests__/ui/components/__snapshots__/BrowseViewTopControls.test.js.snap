// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders corectly 1`] = `
<div
  className="top-controls"
>
  <select
    name="category"
  >
    <option
      value="all"
    >
      All
    </option>
    <option
      value="comedy"
    >
      Comedy
    </option>
    <option
      value="action"
    >
      Action
    </option>
    <option
      value="horror"
    >
      Horror
    </option>
    <option
      value="scifi"
    >
      Sci-Fi
    </option>
    <option
      value="drama"
    >
      Drama
    </option>
  </select>
</div>
`;
